src/instructlab/model/backends/vllm.py

```diff
@@ -120,11 +135,16 @@ def run_vllm(
     vllm_process = None
     vllm_cmd = [sys.executable, "-m", "vllm.entrypoints.openai.api_server"]
 
-    if "--host" not in vllm_args:
-        vllm_cmd.extend(["--host", host])
 
-    if "--port" not in vllm_args:
-        vllm_cmd.extend(["--port", str(port)])
+    # TODO: there should really be a better way to do this, and there probably is
```

Comment:
If there isn't any conditional logic you need to do with these args, then the cleaner way would be to just use a single `.extend` and pass everything in as a single list. Not something I'd block on though.